tools/clang/test/CodeGenHLSL/quick-test/matrix_orientation_pragma_granularity.hlsl

```diff
@@ -0,0 +1,24 @@
+// RUN: %dxc /T vs_6_0 /E main > %s | FileCheck %s
```

Comment:
Could you add some test scenarios surrounding usage of `/Zpr` and `/Zpc`. AFAIK, these flags should override any type attributes defined files using means such as pragma pack_matrix , etc. ?

Reply:
That's not correct, `/Zpr` and `/Zpc` have exactly the same behavior as having a corresponding `#pragma pack_matrix` right before the start of the file (except as far a rewriting goes). Let me see if I can improve a test to reflect this.

Reply:
Actually if you don't mind, I'll do it as a subsequent checkin. I want to update `matrix_orientation_overrides.hlsl` to have `/Zpr` and `/Zpc` flavors, but that file depends on my other pull request.

Reply:
Here you go: #1807